EventSource/src/util/Processing.ts

```diff
@@ -0,0 +1,123 @@
+import { extractTimestampFromLiteral, TREE, turtleStringToStore } from "@treecg/versionawareldesinldp";
+import { Quad, Quad_Subject, Store, DataFactory, Literal } from "n3";
+const namedNode = DataFactory.namedNode;
+import { Resource } from "./EventSource";
+import { existsSync, readFileSync } from "fs";
+
+/**
+ * Returns all the triples as a store from a file at a given filepath
+ * 
+ * @param filepath
+ * @returns {Promise<Store | null>}
+ */
+export async function storeFromFile(filepath: string): Promise<Store | null> {
+    if (existsSync(filepath)) {
+        return await turtleStringToStore(readFileSync(filepath, "utf-8"));
+    }
+    return null
+}
```

Comment:
```suggestion
export async function storeFromFile(filepath: string): Promise<Store> {
    if (existsSync(filepath)) {
        return await turtleStringToStore(readFileSync(filepath, "utf-8"));
    }
    return new Store()
}
```
Does it make sense to have an empty store returned here?

Reply:
I personally feel like passing in an invalid filepath resulting in `null` makes more sense than an empty store (which would be the result of a valid but empty file), but I can change it to return a new store instead if you prefer.